magento/pwa-studio
# [Fix bug]: Fix redirect error for sub-categories

## Description
It gets stuck at loading when trying to open a subcategory that has been redirected.
In the source code, it used history.replace() function but missing the "/" for params.

### Possible solutions
Append "/" to the head of params will fix the error.

## Related Issue
Closes #2860.

## Acceptance

### Verification Stakeholders

### Specification

### Verification Steps
1. Login to admin panel
2. Go to "Marketing / SEO & Search / URL Rewrites"
3. Add an URL rewrite for a subcategory (use 301 or 302 redirect)
4. Go to front-page and clear the browser cache
5. Navigate to the subcategory
6. Make sure the page does not stuck at loading screen.

## Screenshots / Screen Captures (if appropriate)
![image](https://user-images.githubusercontent.com/66080981/98905474-0df66500-24ee-11eb-929c-cb962c478d6e.png)

![image](https://user-images.githubusercontent.com/66080981/98906404-9295b300-24ef-11eb-980b-7c772da3d5c9.png)

## Checklist
* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.


### Resolved issues:
1. [x] resolves magento/pwa-studio#2950: [Fix bug]: Fix redirect error for sub-categories
